binary_wheel_builder/api/build.py

```diff
@@ -14,6 +14,8 @@
 from binary_wheel_builder.wheel.reproducible import ReproducibleWheelFile
 from binary_wheel_builder.wheel.util import generate_metadata_file, generate_wheel_file
 
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
```

Comment:
chore: Please move this to `cli/main.py`

Logging configuration should always happen at the consumer side of things. If this is used as library code for example, this would potentially overwrite logging configuration at runtime, potentially breaking the project configuration.